CMakeLists.txt

```diff
@@ -92,7 +92,7 @@ if (NOT CPACK_GENERATOR)
   elseif(MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "ubuntu" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "debian")
     set(CPACK_GENERATOR DEB CACHE STRING "CPack generator to use, e.g. {DEB, RPM, TGZ}." FORCE)
     message(STATUS "Setting unspecified CPACK_GENERATOR to ${CPACK_GENERATOR}. This is the correct setting for normal builds.")
-  elseif(MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "centos" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "opensuse" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "fedora" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "almalinux")
+  elseif(MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "centos" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "opensuse" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "fedora" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "almalinux" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "rocky")
```

Comment:
consider... the IN_LIST notation?    might be useful here and in other places...

https://stackoverflow.com/a/23323790/6561189

Reply:
nice!